frontend/packages/ux-editor/src/components/config/componentSpecificContent/Summary2/Summary2Component.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import type { IGenericEditComponent } from '../../componentConfig';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import {
+  StudioCard,
+  StudioHeading,
+  StudioNativeSelect,
+  StudioParagraph,
+  StudioTextfield,
+} from '@studio/components';
+import { useFormLayoutsQuery } from '../../../../hooks/queries/useFormLayoutsQuery';
+import { useStudioEnvironmentParams } from 'app-shared/hooks/useStudioEnvironmentParams';
+import { useAppContext, useComponentTypeName } from '../../../../hooks';
+import { useTranslation } from 'react-i18next';
+import { Summmary2ComponentTargetSelector } from './Summary2ComponentTargetSelector';
+import { getAllLayoutComponents } from '../../../../utils/formLayoutUtils';
+
+export const Summary2Component = ({
+  component,
+  handleComponentChange,
+}: IGenericEditComponent<ComponentType.Summary2>) => {
+  const { t } = useTranslation();
+  const { org, app } = useStudioEnvironmentParams();
+  const { selectedFormLayoutSetName, selectedFormLayoutName } = useAppContext();
+  const { data: formLayoutsData } = useFormLayoutsQuery(org, app, selectedFormLayoutSetName);
+  const componentTypeName = useComponentTypeName();
+
+  const excludedComponents = [
+    ComponentType.Summary2,
+    ComponentType.NavigationButtons,
+    ComponentType.NavigationBar,
+  ]; // TODO: Add more components that should be excluded
+
+  const components = Object.values(formLayoutsData).flatMap((layout) =>
+    getAllLayoutComponents(layout, excludedComponents),
+  );
+  const componentOptions = components.map((e) => ({
```

Comment:
Can we give ```e``` a more meaningful name, and also preferably a type? 😄

Reply:
Good point, changed 👍